frontend/packages/pipelines-plugin/src/components/pipelineruns/logs/Logs.tsx

```diff
@@ -29,33 +29,22 @@ const Logs: React.FC<LogsProps> = ({
   const { name } = container;
   const { kind, metadata = {} } = resource;
   const { name: resName, namespace: resNamespace } = metadata;
+  const scrollToRef = React.useRef<HTMLDivElement>(null);
   const contentRef = React.useRef<HTMLDivElement>(null);
   const [error, setError] = React.useState<boolean>(false);
   const resourceStatusRef = React.useRef<string>(resourceStatus);
   const onCompleteRef = React.useRef<(name) => void>();
   onCompleteRef.current = onComplete;
+
   const appendMessage = React.useRef<(blockContent) => void>();
-  const prevFetchNewline = React.useRef(true);
+
   appendMessage.current = React.useCallback(
     (blockContent: string) => {
```

Comment:
Logically, this method should be cleaned up more. Your two if-statements share the same logical check.